po/vi.po

```diff
@@ -2276,84 +2276,92 @@ msgstr ""
 
 #: src/welcome-day-1.md
 msgid "Welcome to Day 1"
-msgstr ""
+msgstr "Chào mừng tới Ngày 1"
 
 #: src/welcome-day-1.md
 msgid ""
 "This is the first day of Rust Fundamentals. We will cover a lot of ground "
 "today:"
 msgstr ""
+"Đây là ngày học đầu tiên của Rust Căn Bản. Sẽ có rất nhiều nội dung mà chúng "
+"ta bao quát trong hôm nay:"
 
 #: src/welcome-day-1.md
 msgid ""
 "Basic Rust syntax: variables, scalar and compound types, enums, structs, "
 "references, functions, and methods."
 msgstr ""
+"Câu lệnh Rust cơ bản: các biến, kiểu dữ liệu đơn giản và phức hợp, enum, "
+"struct, tham chiếu, hàm và phương thức."
 
 #: src/welcome-day-1.md
 msgid "Types and type inference."
-msgstr ""
+msgstr "Các kiểu dữ liệu và suy luận kiểu."
 
 #: src/welcome-day-1.md
 msgid "Control flow constructs: loops, conditionals, and so on."
-msgstr ""
+msgstr "Các cấu trúc điều kiển luồng: vòng lặp, điều kiện, vân vân..."
 
 #: src/welcome-day-1.md
 msgid "User-defined types: structs and enums."
-msgstr ""
+msgstr "Các kiểu dữ liệu tự định nghĩa: struct và enum."
 
 #: src/welcome-day-1.md
 msgid "Pattern matching: destructuring enums, structs, and arrays."
-msgstr ""
+msgstr "So khớp mẫu: hủy enum, struct, và mảng."
 
 #: src/welcome-day-1.md src/welcome-day-2.md src/welcome-day-3.md
 #: src/welcome-day-4.md
 msgid "Schedule"
-msgstr ""
+msgstr "Mục lục"
 
 #: src/welcome-day-1.md src/welcome-day-1-afternoon.md src/welcome-day-2.md
 #: src/welcome-day-2-afternoon.md src/welcome-day-3.md
 #: src/welcome-day-3-afternoon.md src/welcome-day-4.md
 #: src/welcome-day-4-afternoon.md
 msgid "In this session:"
-msgstr ""
+msgstr "Trong buổi này:"
 
 #: src/welcome-day-1.md
 msgid "[Welcome](./welcome-day-1.md) (5 minutes)"
-msgstr ""
+msgstr "[Lời Chào Mừng](./welcome-day-1.md) (5 phút)"
 
 #: src/welcome-day-1.md
 msgid "[Hello, World](./hello-world.md) (15 minutes)"
-msgstr ""
+msgstr "[Hello, World](./hello-world.md) (15 phút)"
 
 #: src/welcome-day-1.md
 msgid "[Types and Values](./types-and-values.md) (40 minutes)"
-msgstr ""
+msgstr "[Kiểu Dữ Liệu Và Giá Trị](./types-and-values.md) (40 phút)"
 
 #: src/welcome-day-1.md
 msgid "[Control Flow Basics](./control-flow-basics.md) (40 minutes)"
-msgstr ""
+msgstr "[Điều Khiển Luồng Cơ Bản](./control-flow-basics.md) (40 phút)"
 
 #: src/welcome-day-1.md
 msgid ""
 "Including 10 minute breaks, this session should take about 2 hours and 5 "
 "minutes"
 msgstr ""
+"Buổi học nên kéo dài khoảng 2 tiếng và 5 phút, bao gồm thời gian 10 phút "
+"nghỉ trưa"
 
 #: src/welcome-day-1.md
 msgid "Please remind the students that:"
-msgstr ""
+msgstr "Hãy lưu ý các học viên rằng:"
 
 #: src/welcome-day-1.md
 msgid ""
 "They should ask questions when they get them, don't save them to the end."
 msgstr ""
+"Các bạn nên giơ tay ngay khi bật ra được câu hỏi nào đó, đừng để dành đến "
```

Comment:
Nit:  "Học viên nên đặt câu hỏi bất kì khi nào thay vì đợi đến cuối buổi học" is more natural.